subworkflows/local/scatter_wipe_gather/main.nf

```diff
@@ -0,0 +1,45 @@
+include { WIPER   } from '../../../modules/local/wipertools/wipe'
+include { SCATTER } from '../../../modules/local/wipertools/scatter'
+include { GATHER  } from '../../../modules/local/wipertools/gather'
+
+
+workflow SCATTER_WIPE_GATHER {
+
+    take:
+    ch_fastq // channel: [ val(meta), [ .fastq ] ]
+
+    main:
+
+    SCATTER {
+        ch_fastq
+    }
+
+    ch_wiper = Channel.empty()
+    ch_wiper = SCATTER.out.fastq_chunks.flatMap { metaData, filePaths -> filePaths instanceof List ? filePaths.collect { files -> [metaData, files] } : [[metaData, filePaths]] }
+
+    WIPER {
+        ch_wiper
+    }
+
+    ch_fastq_gather  = Channel.empty()
+    ch_report_gather = Channel.empty()
+    ch_fastq_gather  = WIPER.out.fixed_fastq.map{ metaData, fastq -> tuple( (fastq.baseName =~ /(.+)_chunk/)[0][1], metaData, fastq ) }
+                                    .groupTuple()
+                                    .map{ basename, metadata, fastq -> tuple(basename, metadata.first(), fastq) }
+    ch_report_gather = WIPER.out.report.map{ metaData, report -> tuple( (report.baseName =~ /(.+)_chunk/)[0][1], metaData, report ) }
+                                    .groupTuple()
+                                    .map{ basename, metadata, report -> tuple(basename, metadata.first(), report) }
+    GATHER(
+        ch_fastq_gather,
+        ch_report_gather
+    )
+
+    ch_versions = Channel.empty()
+    ch_versions = ch_versions.mix(WIPER.out.versions.first())
+
```

Comment:
You need to mix the versions of all modules, not just `WIPER` even if they use the same container (you'll see why once you re-add MultiQC ;) )